portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,86 @@
+
```

Comment:
Nit: there are a couple of issues with the formatting in these html files (e.g. an extra line here, Google's HTML style guide prefers indentation using [2 spaces at a time](https://google.github.io/styleguide/htmlcssguide.html#Indentation), etc). I'll have a look to see if there is a code formatting tool available to you to help with this.